src/main/java/com/google/api/gax/grpc/PageStreamingCallSettings.java

```diff
@@ -59,79 +56,77 @@
   }
 
   /** Package-private, for use by UnaryApiCallable. */
-  UnaryApiCallable<RequestT, PagedListResponse<RequestT, ResponseT, ResourceT>> createPagedVariant(
+  UnaryApiCallable<RequestT, PagedListResponseT> createPagedVariant(
       ManagedChannel channel, ScheduledExecutorService executor) {
     UnaryApiCallable<RequestT, ResponseT> baseCallable = createBaseCallable(channel, executor);
-    return baseCallable.pageStreaming(pageDescriptor);
+    return baseCallable.pageStreaming(pageStreamingFactory);
   }
 
-  public static <RequestT, ResponseT, ResourceT> Builder<RequestT, ResponseT, ResourceT> newBuilder(
-      MethodDescriptor<RequestT, ResponseT> grpcMethodDescriptor,
-      PageStreamingDescriptor<RequestT, ResponseT, ResourceT> pageDescriptor) {
-    return new Builder<>(grpcMethodDescriptor, pageDescriptor);
+  public static <RequestT, ResponseT, PagedListResponseT>
+      Builder<RequestT, ResponseT, PagedListResponseT> newBuilder(
+          MethodDescriptor<RequestT, ResponseT> grpcMethodDescriptor,
+          PageStreamingFactory<RequestT, ResponseT, PagedListResponseT> pageStreamingFactory) {
```

Comment:
I'm confused about how PageStreamingFactory can be package-private when it is a parameter in a public method.